diktat-common/src/test/resources/test-rules-config.yml

```
@@ -0,0 +1,230 @@
+- name: DIKTAT_COMMON
+  enabled: true
+  configuration:
+    isCopyrightMandatory: true
+    domainName: org.cqfn.diktat
+    copyrightText: Copyright (c) Huawei Technologies Co., Ltd. 2012-2020. All rights reserved.
```

```suggestion
    copyrightText: Copyright (c) This is an example. 2012-2020. All rights reserved.
```
Let's make config for tests more generic